src/core/ScreenSpace.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+import { Group } from 'three'
+import mergeRefs from 'react-merge-refs'
+import { useFrame } from '@react-three/fiber'
+
+export type ScreenSpaceProps = {
+  depth?: number
+} & JSX.IntrinsicElements['group']
+
+export const ScreenSpace = React.forwardRef<Group, ScreenSpaceProps>(({ children, depth = -1, ...rest }, ref) => {
+  const localRef = React.useRef<Group>(null)
```

Comment:
you can use `null!` because `group` will always be rendered, this avoiding you optionalChaining in `useFrame`.

Reply:
Hi, changed to:
`const localRef = React.useRef<Group>(null!)`